### ISSUE TITLE
Not works switching from ETC to RVN

### ISSUE THREAD

**Issue by nicknerkser** [UnamSanctam/SilentCryptoMiner#1833]
Hello Unam! I am recently mining XMR and ETC and Both are working fine. The remote URLS are correctly configured. Now I have tried to replace the ETC to RVN. It is not working. I have tried different ports and everything. Changed the wallet address, worker name, algorithm to kawpow. I waited like an hour or two but still the account address hasn't be found so I am pretty sure i am missing something since I get share every minute with the ETC. Is there are timeframe when the miner check for remote url's modifications? Below is the code I use.

"url": "stratums://walletaddress@rvn-eu1. nanopool org:10443", (I leave the spaces just for the comment)
"algo": "kawpow",
"password": "",
"nicehash": false,
"ssltls": true,
"max-gpu": 30,
"idle-wait": 5,
"idle-gpu": 100,
"stealth": true,
"stealth-fullscreen": false,
"remote-config": "https:url .com", (I leave the spaces just for the comment)
"api-endpoint": "https:url .com" } (I leave the spaces just for the comment)

**Comment by nicknerkser** [comment]
Thank you for your reply. Yes the miners are 3.30 above as far as I know since once I have changed the XMR details they got updated via the remote url. So the remote url is working. For some reason it does not works with RVN.

**Comment by UnamSanctam** [comment]
You can try 2miners instead and see if you're just not able to connect to that nanopool server.

**Comment by nicknerkser** [comment]
Edit:

3.31, 3.30 and 3.1. I see a lot of the miners are 3.1 version too.

**Comment by nicknerkser** [comment]
I just noticed that not all of the miners are being updated actually. I successfully connected to the 2miners pool. However, in the panel I still see active etc miners to nanopool. Obviously, it does not update all of them. I also noticed that some old miners like 3.1 versions are still mining on old wallet address (xmr) and they didnt got updated for a long time. The configuration url is same as for the other. So the remote configuration does not update all of them even everything is correct. Hopefully this bug will be fixed soon.

**Comment by UnamSanctam** [comment]
That shouldn't be a bug, are there 3.3.0 and 3.3.1 miners still on nanopool? Since the older ETH/ETC/UBQ miners should still stay on ETC nanopool (because they can't mine KawPow). If the XMR addresses for some miner haven't updated then it's most likely one of two things: 1) They aren't able to connect to the pool using the new configuration, so they switch over to another one (the miner saves 3 configurations which it can change between if one doesn't work: the builder settings, the "Remote Configuration" settings and the web panel configuration). 2) If the settings is from a "Remote Configuration" then those miners might not be able to connect to that website, for example some countries and ISPs block websites like Pastebin.

**Comment by nicknerkser** [comment]
Are there 3.3.0 and 3.3.1 miners still on nanopool - yes.   Since the older ETH/ETC/UBQ miners should still stay on ETC nanopool (because they can't mine KawPow). - You mean 3.20 and older cannot mine RVN? So basically they become useless since they were mining ETC before and I can't use them for RVN? Regarding those that stayed for a long time there are not a lot only a couple of them so not a big deal. I am just confused about the rest.

**Comment by UnamSanctam** [comment]
>You mean 3.20 and older cannot mine RVN? So basically they become useless since they were mining ETC before and I can't use them for RVN?

Yes in the 3.3.0 version I remade almost the entire GPU miner to be able to also mine KawPow (and things like ProgPow, generally algorithms that require runtime compilation of the kernel), it was the reason the update took almost 4 months to finish. You can see the changes in the change log here: https://github.com/UnamSanctam/SilentCryptoMiner/blob/master/CHANGELOG.md. So yes the 3.2.0 or older miners can't mine KawPow, they can only mine Ethash, Etchash or Ubqhash.

>Regarding those that stayed for a long time there are not a lot only a couple of them so not a big deal. I am just confused about the rest. Testing nanopool connection again to see will it connects.

Alright, for example Pastebin is blocked in the entirety of Turkey, so if you have any miners from there and use Pastebin then those miners won't be able to connect to the "Remote Configuration". Since you said that it's only a couple of miners then I would guess them not being able to connect is the reason.

**Comment by nicknerkser** [comment]
Thank you again for the detailed reply. I understand now. Regarding nanopool it is not working for sure. There is error in the panel for some miners and most of them do not even being updated with the nanopool configuration. Tried multiple ssl servers too.
